static/usr/bin/fixrtc

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+# XXX: remove this, once the debugging is done
+if [ -e /run/mnt/ubuntu-seed ]; then
+    LOG=/run/mnt/ubuntu-seed/fixrtc-run.log
+else
+    LOG=/run/fixrtc-run.log
+fi
+exec >> "$LOG"
+exec 2>&1
+
+printf "%s\n" "Starting $(date)"
+TIMESYNC_CLOCK=/var/lib/systemd/timesync/clock
+SELF=$(readlink -f "$0")
+
+# get the mtime of this script
```

Comment:
We should not use mtime of this script, and instead we should use `/usr/lib/clock-epoch` which is always provided in UC20+ initrd. It is the start time of when initrd was generated, and used by systemd to initialize clocks prior to loaded rtc modules causing havoc. No time should  ever be set to older than `/usr/lib/clock-epoch`.

Reply:
Thanks, added this now.